apps/omg_watcher_info/lib/omg_watcher_info/release_tasks/ethereum_tasks/add_eth_height_to_eth_events.ex

```diff
@@ -0,0 +1,106 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.ReleaseTasks.EthereumTasks.AddEthereumHeightToEthEvents do
+  @moduledoc """
+  Prior to https://github.com/omisego/elixir-omg/pull/1509, `eth_height` was not persisted in the Watcher Info DB
+  for `eth_events`. This module will add `eth_height` to all persisted `eth_events` where this value is non-existent.
+  """
+  use Ecto.Migration
+  alias OMG.Eth.Encoding
+  alias OMG.WatcherInfo.DB
+
+  import Ecto.Query, only: [from: 2]
+
+  @max_db_rows 100
+  @max_eth_requests 25
+
+  def run() do
+    DB.Repo.transaction(fn ->
+      stream_events_from_db()
+      |> stream_create_requests()
+      |> stream_batch_requests()
+      |> stream_make_requests()
+      |> stream_concatenate_responses()
+      |> stream_format_responses()
+      |> Enum.map(&update_record/1)
+    end)
+  end
+
+  def stream_events_from_db() do
```

Comment:
mixing public and private functions. public on top, private to the bottom.

Reply:
Done e5513b8.